src/compositemark/common.ts

```diff
@@ -121,6 +127,34 @@ export function makeCompositeAggregatePartFactory<P extends PartsMixins<any>>(
   };
 }
 
+export function compositeMarkExtractTransformsFromEncoding(
```

Comment:
Why do we need a new method?

Why shouldn't this format replacement be down in `extractTransformsFromEncoding`?